nebulae/default/names.dat

```diff
@@ -402,7 +403,7 @@ NGC  3372            _("Homunculus Nebula") # WK, WSO
 NGC  3372            _("Keyhole Nebula") # WK, SIMBAD, BCH, SG, WSO, DN, DAN
 NGC  3372            _("The Great Nebula in Carina") # DN, APOD
 NGC  3432            _("Knitting Needle Galaxy") # B500, SD
-NGC  3521            _("Bubble Galaxy") # 
+NGC  3521            _("Bubble Galaxy") # AAPOD2
```

Comment:
Please add ESA/Hubble source as primary also

Reply:
no abbrevation code for ESA/Hubble source?

Reply:
Well, ESA/Hubble source is similar to APOD and AAPOD2, so, you need introduce new abbreviation for it (probably scheme will be similar to APOD)

Reply:
ok